pages/community-lessons.md

```diff
@@ -1,145 +0,0 @@
----
-layout: default
-title: "Community Developed Lessons"
```

Comment:
I'm not so keen on kicking this out. I think there is actually a lot of value in this (in future). For example, what about all the lessons from Code Refinery, wouldn't it be nice if we listed them on our website somewhere. We can probably find the Amy code to get the repo information and be able to keep this up to date using a scheduled Action build.

Reply:
Restored by 333a4a9e.